python/ray/tune/trainable.py

```diff
@@ -119,13 +119,11 @@ def default_resource_request(cls, config):
             >>>         extra_cpu=config["workers"],
             >>>         extra_gpu=int(config["use_gpu"]) * config["workers"])
         """
-
         return None
 
     @classmethod
     def resource_help(cls, config):
         """Returns a help string for configuring this trainable's resources."""
-
```

Comment:
Standard google docstrings should have a newline after the docstring iirc

Reply:
i think that just refers to if the docstring is > 1 line the first line has to have a newline after before the rest of the doc (ie within the str). there are multiple examples with no newline after, eg: http://google.github.io/styleguide/pyguide.html#244-decision.